### PR TITLE
Add C# example to CPU optimization

### PR DESCRIPTION
**Pull request by ShawnHardern**
Add C# examples to CPU optimization

### CODE REVIEW

**Review by raulsntos — APPROVED** [review]
LGTM.

File: tutorials/performance/cpu_optimization.rst [hunk 1]
```diff
@@ -102,6 +103,16 @@ the following:
     var time_end = Time.get_ticks_usec()
     print("update_enemies() took %d microseconds" % time_end - time_start)
 
+ .. code-tab:: csharp
+
+    var timeStart = Time.GetTicksUsec();
+
+    // Your function you want to time
```
Comment (AThousandShips) [inline tutorials/performance/cpu_optimization.rst:110]:
```suggestion
    // Your function you want to time.
```

**Review by AThousandShips — APPROVED** [review]

**Comment by AThousandShips** [comment]
Thanks!

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.3 in https://github.com/godotengine/godot-docs/pull/10038.

**Pull request merged**